samples/python/topology/spl/files.py

```diff
@@ -38,7 +38,7 @@ def main():
     lines.print()
     
     # Now execute the topology by submitting to a standalone context.
-    streamsx.topology.context.submit("STANDALONE", topo)
+    streamsx.topology.context.submit("TOOLKIT", topo)
```

Comment:
Leave as standalone.

Reply:
Sorry, I created the pull request by mistake before finishing my review.  I will change this.